internal/commands/run.go

```diff
@@ -105,6 +111,7 @@ func newRunCmd() *cobra.Command {
 		Args:  cobra.MaximumNArgs(1),
 	}
 
+	cmd.PersistentFlags().BoolVar(&dirty, "dirty", false, "mount project directory in read-write mode")
```

Comment:
Maybe add description of both cases:

```suggestion
	cmd.PersistentFlags().BoolVar(&dirty, "dirty", false, "if true mounts project directory in read-write mode. otherwise copies files respecting .gitignore")
```

Reply:
See https://github.com/cirruslabs/cirrus-cli/pull/34/files#r471635981.